lib/krane/cluster_resource_discovery.rb

```diff
@@ -66,7 +76,7 @@ def api_paths
 
     def fetch_api_path(path)
       @api_path_cache[path] ||= begin
-        raw_json, err, st = kubectl.run("get", "--raw", path, attempts: 2, use_namespace: false)
+        raw_json, err, st = kubectl.run("get", "--raw", File.join(cluster_url, path), attempts: 2, use_namespace: false)
```

Comment:
I've changed this again to just use `path`. Presumably your kubeconfig should already have the base URI. @strowi when you test this can you let me know if this fails; unfortunately I can't easily test against your situation